internal/cmd/passwords/passwords.go

```diff
@@ -0,0 +1,121 @@
+package passwords
+
+import (
+	"encoding/json"
+	"strings"
+	"time"
+
+	"github.com/lensesio/tableprinter"
+	"github.com/planetscale/cli/internal/cmdutil"
+	"github.com/spf13/cobra"
+
+	ps "github.com/planetscale/planetscale-go/planetscale"
+)
+
+// PasswordCmd handles branch passwords.
+func PasswordCmd(ch *cmdutil.Helper) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "password <command>",
+		Short:             "Create, list, show, and delete branch passwords",
+		PersistentPreRunE: cmdutil.CheckAuthentication(ch.Config),
+	}
+
+	cmd.PersistentFlags().StringVar(&ch.Config.Organization, "org", ch.Config.Organization, "The organization for the current user")
+	cmd.MarkPersistentFlagRequired("org") // nolint:errcheck
+
+	cmd.AddCommand(CreateCmd(ch))
+	cmd.AddCommand(ListCmd(ch))
+	cmd.AddCommand(DeleteCmd(ch))
+	// cmd.AddCommand(ShowCmd(ch))
```

Comment:
Do you plan to add this command or should we remove this line? The description also says:

```
Create, list, show, and delete branch passwords
```

Reply:
I'll remove it.